sdcm/utils/common.py

```diff
@@ -1179,6 +1179,16 @@ def get_tables_columns_list(entity_type):
         for ks_cf, column_types in list(avaialable_ks_cf.items()):
             if 'counter' in column_types:
                 avaialable_ks_cf.pop(ks_cf)
+
+    if filter_empty_tables:
+        for ks_cf in list(avaialable_ks_cf.keys()):
+            cmd = f'SELECT * FROM {ks_cf} LIMIT 1'
+            # TODO: make this work with cql python driver, but would need to refactor the session of of tester class
+            result = loader_node.run_cqlsh(cmd=cmd, timeout=request_timeout, verbose=False, target_db_node=db_node,
```

Comment:
May be it's better to check if the data files exist? Because of even the data is queried, but wasn't flushed to the disk, it still will be "empty table"

Reply:
so maybe that on the user of the this list, to check existence of files before ?

Reply:
What do you mean, which user?

Reply:
caller, whom calls this function, should do the check of existence of files. and skip to the next table.

also if this return empty list, nemesis should skip, and not fail.

Reply:
@juliayakovlev, I think this PR would minimize the issues we are seeing, but more work on the nemesis end is needed to handle no tables returned from this function, and the case you mentioned, when table doesn't have sstables ready, the nemesis should skip to the next table (not fail)

Reply:
I think every nemesis could decide how to behave in this case: fails or not